eng/pipelines/templates/steps/run_apistub.yml

```diff
@@ -8,17 +8,18 @@ parameters:
 
 # Please use `$(TargetingString)` to refer to the python packages glob string. This was previously `${{ parameters.BuildTargetingString }}`.
 steps:
-  - task: PythonScript@0
-    condition: and(succeededOrFailed(), ne(variables['Skip.ApiStubGen'],'true'))
-    displayName: 'Generate API stub files'
-    inputs:
-      scriptPath: 'scripts/devops_tasks/setup_execute_tests.py'
-      arguments: >-
-        "$(TargetingString)"
-        ${{ parameters.AdditionalTestArgs }}
-        -d "$(Build.ArtifactStagingDirectory)"
-        --mark_arg="${{ parameters.TestMarkArgument }}" 
-        --service="${{ parameters.ServiceDirectory }}" 
-        --toxenv="apistub"
-        --disablecov
-        --filter-type="Omit_management"
\ No newline at end of file
+  - bash: |
+      python -m venv $(Agent.BuildDirectory)/venv
+      source $(Agent.BuildDirectory)/venv/bin/activate
+      mkdir -p $(Agent.BuildDirectory)/workdir
+      
+      python -m pip install -r $(Build.SourcesDirectory)/eng/apiview_reqs.txt --index-url="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-python/pypi/simple/"
+    displayName: Install API Stub Generator
+
+  - bash: |
+      source $(Agent.BuildDirectory)/venv/bin/activate
+      
+      python -m pip freeze
+
+      python $(Build.SourcesDirectory)/scripts/devops_tasks/invoke_apistub.py $(TargetingString) --service=${{ parameters.ServiceDirectory }} --workdir=$(Agent.BuildDirectory)/workdir
+    displayName: Install API Stub Generator
```

Comment:
These job names are the same. Do you want to just merge them, or keep the names separate?

Reply:
I'm still messing about with it, it'll be fixed. Thanks for the heads up.